sdk/core/core-amqp/src/errors.ts

```diff
@@ -629,6 +629,19 @@ function isBrowserWebsocketError(err: any): boolean {
   return result;
 }
 
+/**
+ * @internal
+ * Checks whether a object is an ErrorEvent or not. https://html.spec.whatwg.org/multipage/webappapis.html#errorevent
+ * @param err - object that may contain error information
+ */
+function isErrorEvent(err: any): err is { message: string; error: any } {
+  let result: boolean = false;
+  if (typeof err.error === "object" && typeof err.message === "string") {
+    result = true;
+  }
+  return result;
```

Comment:
nitpick: couldn't it be simplified as follows?
```suggestion
  return typeof err.error === "object" && typeof err.message === "string";
```